### ISSUE TITLE
Highlighting of matched text

### ISSUE THREAD

**Issue by dlong500** [arendjr/selectivity#40]
I've search through the docs and issues and don't see any mention of being able to highlight matched text in the results list.  Am I missing something?

The best option in my opinion would be to wrap matching text with a span element that could be styled in the CSS file.

**Comment by shoaibshakeel381** [comment]
Can you please explain what do you mean by `highlighting` here. `selectivity` only shows elements matching search term. So there is no need to highlight anyone of them.

**Comment by arendjr** [comment]
Hi, there's no support for such highlighting (indeed, you mean the highlighting like shoaibshakeel381 illustrates, right?) yet. PR would be welcome :)

**Comment by dlong500** [comment]
Yes, that's exactly what I mean.  So using the above example there would be something like the below generated for the item html:
`<span class="selectivity-matched-text">A</span>n <span class="selectivity-matched-text">A</span>pple`

I'd love to submit a PR but I don't have the time at the moment.  Maybe in a few weeks if no one else can tackle it.

**Comment by shoaibshakeel381** [comment]
Well as a hack you can pass you `term` in results to `callback` as additional property. And in you template where you get single item you'll also get your term as property. Then use this proper to search for each occurrence of term in label, split label from there (length will be search term), and enclose it inside a styled span. After that concatenate you string again and return html.
